src/amo/components/ErrorPage/NotFound/index.js

```diff
@@ -7,7 +7,7 @@ import Link from 'amo/components/Link';
 import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
 import translate from 'core/i18n/translate';
 import { sanitizeHTML, visibleAddonType } from 'core/utils';
-import { getLocalizedTextWithLinkParts } from 'core/utils/i18n';
+import { replaceStringsWithJSX } from 'core/utils/i18n';
```

Comment:
Whoops, this module was [added by mistake](https://github.com/mozilla/addons-frontend/commit/d43a84130a#diff-9b5b84331c178ffca61a6571cde8e558). It should have gone in [`core/i18n/utils`](https://github.com/mozilla/addons-frontend/blob/56da66feebedbfdb5cfcd74f742c6e81c46515b9/src/core/i18n/utils.js). Can you move the new function there?